src/stores/cloud_environment/mock-data.js

```diff
@@ -0,0 +1,39 @@
+export default {
```

Comment:
No need create this file, you can use `config/runtimes.js`

Reply:
ok